lib/mix/lib/mix/task.ex

```diff
@@ -193,7 +193,10 @@ defmodule Mix.Task do
   end
 
   def alias?(task) when is_atom(task) do
-    Mix.Project.config[:aliases][task]
```

Comment:
Ah, right thanks!  I think I was confused thinking that we were referring to using `!= nil` on the assertions.  

I do like either `!= nil` or `Keyword.has_key?` for being more descriptive.  Is there an advantage to using `Keyword.has_key`?